### DISCUSSION TITLE
po-page-dynamic-table

### DISCUSSION THREAD

**Discussion by wsteixeira** [po-ui/po-angular#1026] [Angular] [answered]
Não estou conseguindo usar "mask" no po-page-dynamic-table!

  readonly fields: Array<PoPageDynamicTableFilters> = [
    { property: 'id', label: 'CPF', key: true, mask: '999.999.999-99' },  
    { property: 'name', label: 'Nome' },  
    { property: 'cardId', label: 'Numero do Cartão', mask: '9999.9999.9999.9999' }
  ];

**Comment by jhonyeduardo** [comment, marked as answer]
Bom dia @wsteixeira 

No DynamicTable a mascara será aplicada apenas se for filtro da busca avançada, ex: 
```
{ property: 'cardId', label: 'Numero do Cartão', mask: '9999.9999.9999.9999', filter: true }
```

https://stackblitz.com/edit/po-ui-6nh2dh?file=src%2Fapp%2Fapp.component.ts
